### DISCUSSION TITLE
BHoM: Re-define properties of BHoMObject

### DISCUSSION THREAD

**Discussion by IsakNaslundBh** [BHoM/BHoM#475] [General] [answered]
`BHoMObject` shold be redefined to have the following properties:

- `Guid`
- `string Name`
- `Fragments List<IFragment>`

CustomData would now be stored as a Fragment. Same things for Tags.

Depending on https://github.com/BHoM/BHoM/issues/476

**Comment by FraserGreenroyd** [comment, marked as answer]
Have converted this to a discussion and wonder how this works in relation with #325 

If we're happy that this isn't happening anymore @IsakNaslundBh feel free to close the discussion :smile:

**Reply by IsakNaslundBh** [reply]
Not sure it was even my idea, or if I just raised the issue after a meeting.

I am personally fine with keeping the CustomData and Tags on the BHoMObject itself. See no real benefit of moving any of them into the fragments. And I think renaming of the GUID property has been discussed and dismissed elsewhere, so, will close.

@adecler @al-fisher feel free to reopen if you disagree

**Reply by al-fisher** [reply]
Agreed @IsakNaslundBh 
Thanks @FraserGreenroyd
